### ISSUE TITLE
[keymgr] cfgen stuck with 0 when init is programmed to 1 after initialization

### ISSUE THREAD

**Issue by weicaiyang** [lowRISC/opentitan#4411]
@tjaychen 

Looks like `init` isn't ignored as expected when initialization is done. It's because `init` will clear `cfgen`

```
keymgr_cfg_en u_cfgen (
    .clk_i,
    .rst_ni,
    .en_i(lc_i.keymgr_en),
    .set_i(op_done | init_done),
    .clr_i(reg2hw.control.start.q | reg2hw.control.init.q),
    .out_o(hw2reg.cfgen.d)
  );
```

Waveform:
/edascratch/weicai-opentitan/ot/keymgr.sim.vcs/km_scb/0.keymgr_cfgen/out_Wed.12.02.20__09.53.39PM
verdi -ssr Verdi.ses &

**Comment by tjaychen** [comment]
o damn, good catch. 
I'll have a look at this.

**Comment by tjaychen** [comment]
o is there any way for me to run the same test? :)

**Comment by weicaiyang** [comment]
Need to get this PR #4418 and run below test
$REPO_PATH/util/dvsim/dvsim.py keymgr_sim_cfg.hjson -i keymgr_cfgen -s 3178879435 -w

There are some other errors, too. Need to solve them with updated scoreboard. You can just check the cfgen behavior in waveform.

**Comment by weicaiyang** [comment]
Fixed via #4420
